src/execution_plan/ops/op_unwind.c

```diff
@@ -8,11 +8,16 @@
 #include "op_unwind.h"
 #include "../../util/arr.h"
 #include "../../arithmetic/arithmetic_expression.h"
+#include "limits.h"
 
-OpBase *NewUnwindOp(uint record_idx, AR_ExpNode **exps) {
+#define UNDEFIND UINT_MAX
+
+OpBase *NewUnwindOp(uint record_idx, AR_ExpNode *exps) {
 	OpUnwind *unwind = malloc(sizeof(OpUnwind));
-	unwind->expIdx = 0;
+	unwind->listIdx = UNDEFIND;
```

Comment:
simply set to -1.

Reply:
it is easier to check `if(op->listIdx < array_len(op->list.array))`